core/src/it/scala/org/ensime/fixture/ProjectFixture.scala

```diff
@@ -3,12 +3,11 @@
 package org.ensime.fixture
 
 import akka.actor.{ ActorRef, ActorSystem }
-import org.scalatest._
 import akka.testkit._
-
 import org.ensime.api._
```

Comment:
could you please back out the import rearrangements?

Probably you need to change your IntelliJ sorting options so they go

```
java.

scala.

<other>
```

I'm not sure how you do that